gift_compassion/models/sponsorship_gift.py

```diff
@@ -516,6 +517,16 @@ def action_verify(self):
         return True
 
     @api.multi
+    def action_in_progress(self):
+        self.write({'state': 'In Progress'})
```

Comment:
Actually we just discussed that in the sprint review. It's wrong to send the message a second time. Then an easier solution is to put the move in draft on suspension and just confirm it on reactivation.
See the jira task was modified accordingly...